cms/static/cms/js/modules/cms.sideframe.js

```diff
@@ -389,6 +414,78 @@ var CMS = window.CMS || {};
                     .off(this.pointerUp)
                     .off(this.pointerMove)
                     .removeAttr('data-touch-action');
+            },
+
+            /**
+             * Retreives the history states from `this.history`.
+             *
+             * @method _getHistory
+             * @private
+             */
+            _getHistory: function _getHistory(type) {
```

Comment:
this should be more like `_moveInHistory` or smth, it doesn't really "get" you anything